packages/cyberstorm/src/components/Avatar/Avatar.module.css

```diff
@@ -1,19 +1,31 @@
 .root {
+  display: flex;
   align-items: center;
   justify-content: center;
-  border-radius: var(--size) !important;
-  background-color: #fff;
+  border-radius: calc(var(--size) * 4.92) !important;
```

Comment:
What's the purpose of oddly specific 4.92? Is it something we want to use here or everywhere with round elements? Should it be (global) variable?

Reply:
It's a common multiplier between the different sizes border-radius'. A bit rough, but will do.